### PR TITLE
Add v1.6 proposals and metadata

### PR DESCRIPTION
**Pull request by movekevin**

### CODE REVIEW

**Review by runtian-zhou — APPROVED** [review]

File: metadata/v1.6/step_3_enable_aptos_unique_identifiers.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable Aptos Unique Identifiers",
+  "description": "AIP-36: Support for aptos unique identifiers (generated using native function generate_unique_address)",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```
Comment (perryjrandall) [inline metadata/v1.6/step_3_enable_aptos_unique_identifiers.json:4]: should this point to the PR?

File: metadata/v1.6/step_1_upgrade_framework.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.6",
+  "description": "This includes changes outlined in https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.6",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```
Comment (perryjrandall) [inline metadata/v1.6/step_1_upgrade_framework.json:4]: should this point to the code at the revision of the releas ebranch?

**Review by perryjrandall — APPROVED** [review]

File: metadata/v1.6/step_1_upgrade_framework.json (hunk 2 above)
Comment (sherry-x) [reply metadata/v1.6/step_1_upgrade_framework.json:4]: this should be the source code for the .mv code in this repo

File: metadata/v1.6/step_1_upgrade_framework.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.6",
+  "description": "This includes changes outlined in https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.6",
```
Comment (sherry-x) [inline metadata/v1.6/step_1_upgrade_framework.json:3]: this is not a valid url, you need to use 1.6.0

Comment (movekevin) [reply metadata/v1.6/step_1_upgrade_framework.json:3]: This is all generated by the builder. Let me fix

File: metadata/v1.6/step_1_upgrade_framework.json (hunk 2 above)
Comment (movekevin) [reply metadata/v1.6/step_1_upgrade_framework.json]: Updated

File: metadata/v1.6/step_1_upgrade_framework.json (hunk 3 above)
Comment (movekevin) [reply metadata/v1.6/step_1_upgrade_framework.json]: Updated

File: metadata/v1.6/step_3_enable_aptos_unique_identifiers.json (hunk 1 above)
Comment (movekevin) [reply metadata/v1.6/step_3_enable_aptos_unique_identifiers.json]: Updated

**Review by sherry-x — APPROVED** [review]

**Pull request closed without merging**